proxy/proxy_test.go

```diff
@@ -23,18 +23,19 @@ import (
 	"path/filepath"
 	"testing"
 
-	"github.com/katzenpost/server_plugins/grpc_plugins/currency/common"
-	"github.com/katzenpost/server_plugins/grpc_plugins/currency/config"
+	"github.com/hashcloak/Meson/common"
+	"github.com/hashcloak/Meson/config"
 	"github.com/stretchr/testify/assert"
 )
 
 const (
-	zcashSendVersion = 0
+	ethSendVersion = 0
 )
 
-type zcashSendRequest struct {
+type ethSendRequest struct {
```

Comment:
Again, I wonder if defining a new structure in the tests is a good practice. I guess it depends on the case, but usually we want to test data structures and logic which are defined in the code itself. (although I don't see where this type is used here)

Reply:
It's actually not even used in the test code anywhere. Maybe it should be removed?

Reply:
Yes, removing sounds good!